src/assets/styles/_type.scss

```diff
@@ -22,3 +20,20 @@ $base-font-size: 14px !default;
   font-size: var(--calcite-app-base-font-size);
   line-height: var(--calcite-app-line-height);
 }
+
+h1.heading {
+  font-size: var(--calcite-app-font-size-3);
+  font-weight: var(--calcite-app-font-weight);
```

Comment:
question: Do we need different font weights?

Reply:
Meaning for the headings? Or in general?

Reply:
In general.

Reply:
Yeah. We tried to make the design work with only the regular (400) font weight, but it wasn't enough contrast.

As a note, I am updating so that all headings get the demi weight, and am nixing the redundant `--calcite-app-font-weight-heading` var.